website/docs/docs/use-dbt-semantic-layer/quickstart-sl.md

```diff
@@ -1,36 +1,197 @@
+:::tip
+
+If you're familiar with writing SQL, you can think of dimensions as the columns you would group by and measures as the columns you would aggregate.
+```sql
+select
+  metric_time_day,  -- time
+  country,  -- categorical dimension
+  sum(revenue_usd) -- measure
+from
+  snowflake.fact_transactions  -- sql table
+group by metric_time_day, country  -- dimensions
+  ```
+:::
+
+## Define metrics
+
+Now that you've created your first semantic model, it's time to define your first metric. MetricFlow supports different metric types like [simple](/docs/build/simple), [ratio](/docs/build/ratio), [cumulative](/docs/build/cumulative), and [derived](/docs/build/derived). You can define metrics in the same YAML files as your semantic models, or create a new file.
+
+The example metric we'll create is a simple metric that refers directly to a measure, based on the `transaction_amount_usd` measure, which will be implemented as a `sum()` function in SQL.
+
+```yaml
+---
+metrics:
+  - name: transaction_amount_usd
+    type: simple
+    type_params:
+    measure: transaction_amount_usd
+```
+ 
+1. Click **Save** and then **Preview** the code in the dbt Cloud IDE.
+2. Run `mf query --metrics <metric_name> --group-by <dimension_name>` to manually query the metrics and dimensions in the IDE.
+3. Run `mf validate-configs` to validate the changes before committing them.
+4. Commit and merge the code changes that contain the metric definitions.
+
+To continue building out your metrics based on your organization's needs, refer to the [Build your metrics](/docs/build/build-metrics-intro) for detailed info on how to define different metric types and semantic models.
+
+## Run a production job
+
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
+
+1. Go to **Deploy** in the navigation header
+2. Select **Jobs** to re-run the job with the most recent code in the deployment environment.
+3. Your metric should appear as a red node in the dbt Cloud IDE and dbt directed acyclic graphs (DAG).
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/metrics_red_nodes.png" width="85%" title="DAG with metrics appearing as a red node" />
+
+
+<details>
+
+<summary>What’s happening internally?</summary>
+- Merging the code into your main branch allows dbt Cloud to pull those changes and builds the definition in the manifest produced by the run. <br />
+- Re-running the job in the deployment environment helps materialize the models, which the metrics depend on, in the data platform. It also makes sure that the manifest is up to date.<br />
+- The Semantic Layer APIs pulls in the most recent manifest and allows your integration information to extract metadata from it.
+</details>
+
+## Set up dbt Semantic Layer
+
+import SlSetUp from '/snippets/_new-sl-setup.md';  
+
+<SlSetUp/>
+
+
+## Connect to the APIs
```

Comment:
I think Connect/Query are probably all one fell swoop; so after Setup, we can say "Connect and Query"

I wonder if we can provide an example with a neutral tool like DataBricks. Or maybe we can use Mode or Hex.. let me think about this

Reply:
no worries! that would be great and easy enough to call it out that we're using 'name_of_integration" for the quickstart

Reply:
ok so we dont have docs from mode or hex yet and hoping  to get them friday